src/server/ApiRoutes/Bib.js

```diff
@@ -47,6 +47,23 @@ export const addHoldingDefinition = (holding) => {
     .filter((data) => data.definition);
 };
 
+const appendDimensionsToExtent = (bib) => {
+  if (bib.extent) {
+    // Check if extent was cataloged with a semicolon already at the end
+    const semicolon = bib.extent.slice(-2) === '; ' || bib.extent.slice(-1) === ';'
```

Comment:
When using the [mockBib](https://github.com/NYPL/discovery-front-end/pull/1891/files#diff-a0e8cba60ce3a8ebdfcdfda3244a76ad53fe222a5e99fad361f2f8605a041d00R13-R14) `extent` is an array or items. `Semicolon` will always be false when extent is an array since [slicing an array returns an array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice).